src/dao/hooks/types.ts

```diff
@@ -205,3 +206,13 @@ export interface UpdateAmountEventData {
   user: string;
   idOrAmount: number;
 }
+export interface DAOConfigDetails {
```

Comment:
If these interfaces are unique to the `useFetchDAOConfig` hook, move them to the top of that file.

Reply:
Done.